src/Servers/Kestrel/Core/src/KestrelServerOptions.cs

```diff
@@ -97,6 +99,7 @@ public class KestrelServerOptions
         /// <remarks>
         /// Defaults to false.
         /// </remarks>
+        [Obsolete($"This property is obsolete and will be removed in a future version. It no longer has any impact on runtime behavior. Use {nameof(Microsoft.AspNetCore.Server.Kestrel.Core.ListenOptions)}.{nameof(Microsoft.AspNetCore.Server.Kestrel.Core.ListenOptions.AddAltSvcHeader)} to configure \"Alt-Svc\" behavior.")]
```

Comment:
API Review: Make this be an error (https://docs.microsoft.com/en-us/dotnet/api/system.obsoleteattribute.-ctor?view=net-5.0#System_ObsoleteAttribute__ctor_System_String_System_Boolean_)